compose.yaml

```diff
@@ -0,0 +1,115 @@
+secrets:
+  # These secrets are used by Authelia
+  JWT_SECRET:
+    file: ${SECRETS_DIR}/JWT_SECRET
+  SESSION_SECRET:
+    file: ${SECRETS_DIR}/SESSION_SECRET
+  STORAGE_ENCRYPTION_KEY:
+    file: ${SECRETS_DIR}/STORAGE_ENCRYPTION_KEY
+  # These secrets are for using Authelia as an OIDC provider
+  HMAC_SECRET:
+    file: ${SECRETS_DIR}/HMAC_SECRET
+  JWT_PRIVATE_KEY:
+    file: ${SECRETS_DIR}/certs/private.pem
+  DEX_CLIENT_SECRET_DIGEST:
+    file: ${SECRETS_DIR}/DEX_CLIENT_SECRET_DIGEST
+
+services:
+  traefik:
+    container_name: traefik
+    image: traefik:latest
+    ports:
+      # HTTP Ports
+      - "80:80"
+      - "443:443"
+      # The Web UI (enabled by --api.insecure=true)
+      - "8080:8080"
+      - "8082:8082"
+    command:
+      - '--providers.file.filename=/settings/traefik.yaml'
+      - '--entryPoints.web.address=:80'
+    volumes:
+      - ./config/traefik.yaml:/settings/traefik.yaml
+      - ./config/secrets/acme_certificates:/acme
+
+  grist:
+    container_name: grist
+    image: gristlabs/grist:latest
+    environment:
+      APP_HOME_URL: ${APP_HOME_URL}
+
+      GRIST_OIDC_IDP_ISSUER: '${APP_HOME_URL}/dex'
+      GRIST_OIDC_IDP_CLIENT_ID: '${GRIST_OIDC_IDP_CLIENT_ID}'
+      GRIST_OIDC_IDP_CLIENT_SECRET: '${GRIST_OIDC_IDP_CLIENT_SECRET}'
+      GRIST_OIDC_IDP_END_SESSION_ENDPOINT: '${APP_HOME_URL}/signed-out'
+
+      # Working with multiple teams is possible but a little harder to
+      # explain and understand, and the UI has rough edges.
+      GRIST_SINGLE_ORG: ${TEAM}
+      # Default email for the "Admin" account
+      GRIST_DEFAULT_EMAIL: ${DEFAULT_EMAIL:-test@example.org}
+    restart: always
+    volumes:
+      # Where to store persistent data, such as documents.
+      - ${PERSIST_DIR}/grist:/persist
+    depends_on:
+      traefik:
+        condition: service_started
+      dex:
+        condition: service_healthy
+    links:
+      - "traefik:${GRIST_DOMAIN}"
+  
+  dex:
+    restart: always
+    container_name: dex
+    image: dexidp/dex:latest
+    volumes:
+      - ./config/dex.yaml:/etc/dex/config.docker.yaml
+      - ./${PERSIST_DIR}/auth/:/persist/auth
```

Comment:
This is where the dex.db database should be mounted, but currently has permissions issues.

Reply:
Okay, I've handled this by passing in the `$UID/$GID` combo from the host into the container parameters.